docs/sources/tempo/traces.md

```diff
@@ -65,11 +65,18 @@ Logs provide an audit trail of activity that create an informational context. Tr
 
 ## Traces versus metrics and logs
 
-Let's say a server takes too long to send data and triggers an alert because that time range is outside of an acceptable threshold. The metrics for those alert help you identify problems and patterns.
+Each observability signal plays a unique role in providing insights into your systems.
+Metrics act as the high-level indicators of system health.
+They alert you that something is wrong or deviating from the norm.
+Logs then help you understand what exactly is going wrong, for example, the nature or cause of the elevated error rates you're seeing in your metrics.
+Traces illustrate where in the sequence of events something is going wrong.
+They let you pinpoint which service in the many services that any given request traverses is the source of the delay or the error.
 
-Traces help you pin-point where in your system or services the problem happens. Traces provide context around the issue you're investigating. For example, traces can pinpoint the specific service where the error started and how that error moved through the system.
+Let's say a server takes too long to send data. Your metrics that track the latency of your system will increase, and they may then trigger an alert once that latency rises outside of an acceptable threshold.
```

Comment:
nit: First person. Should we have exceptions for the collective including the reader?

Reply:
I'll add it to the style roundtable doc :)